scroll.go

```diff
@@ -36,6 +36,7 @@ type ScrollService struct {
 	ignoreUnavailable *bool
 	allowNoIndices    *bool
 	expandWildcards   string
+	headers           map[string]string
```

Comment:
Can we make this a [`http.Header`](https://golang.org/pkg/net/http/#Header) instead.